SimPEG/Survey.py

```diff
@@ -120,7 +121,7 @@ def getP(self, mesh, timeMesh):
         return P
 
 
-class BaseSrc(object):
+class BaseSrc(Props.BaseSimPEG):
```

Comment:
@sgkang this should allow you to put the properties on the source as you wanted to in the SP inversions.